### ISSUE TITLE
ChestTracker Unable to Write NBT Files

### ISSUE THREAD

**Issue by pumrum** [JackFred2/ChestTracker#170]
1. Minecraft Java on Windows 11
2. Fresh install of MC 1.21
3. Fresh install of fabric-installer-1.0.1.exe
4. Added only these mod JARs:
5. chesttracker-2.4.6+1.21.jar | fabric-api-0.100.1+1.21.jar | Searchables-fabric-1.21-1.0.1.jar | YetAnotherConfigLib-3.5.0+1.21-fabric.jar

Connecting to a bukkit/spigot/paper server running 1.21, chesttracker is not persisting inventories between restarts of the game. latest.log includes repeats of:
```
[16:33:35] [Render thread/ERROR]: Error saving object
java.io.IOException: Error encoding to NBT DataResult.Error['Can't access registry ResourceKey[minecraft:root / minecraft:enchantment]
```

[full latest.log shared here](https://pastebin.com/0S3XhWKd)
only the hostname, port, and username have been changed for privacy

[mc debug info in case it's useful](https://imgur.com/a/yYdAXA6)

**Comment by JackFred2** [comment]
Fixed in 2.4.7

**Comment by pumrum** [comment]
thank you so much for the quick fix! as a bonus, I was able to restore my pre-1.21 nbt file that had been marked as "corrupted" and it no longer gets punted by the mod. all is right in the world
